pennylane/labs/resource_estimation/ops/qubit/parametric_ops_single_qubit.py

```diff
@@ -112,17 +163,33 @@ def resource_params(self) -> dict:
     def resource_rep(cls) -> re.CompressedResourceOp:
         return re.CompressedResourceOp(cls, {})
 
+    @classmethod
+    def adjoint_resource_decomp(cls) -> Dict[re.CompressedResourceOp, int]:
+        return {cls.resource_rep(): 1}
+
+    @staticmethod
+    def controlled_resource_decomp(
+        num_ctrl_wires, num_ctrl_values, num_work_wires
+    ) -> Dict[re.CompressedResourceOp, int]:
+        if num_ctrl_wires == 1 and num_ctrl_values == 0 and num_work_wires == 0:
+            return {re.ResourceCRZ.resource_rep(): 1}
+
+        raise re.ResourcesNotDefined
+
+    @classmethod
+    def pow_resource_decomp(cls, z) -> Dict[re.CompressedResourceOp, int]:
+        return {cls.resource_rep(): 1}
+
 
 class ResourceRot(qml.Rot, re.ResourceOperator):
     """Resource class for the Rot gate."""
 
     @staticmethod
-    def _resource_decomp() -> Dict[re.CompressedResourceOp, int]:
-        rx = ResourceRX.resource_rep()
+    def _resource_decomp(**kwargs) -> Dict[re.CompressedResourceOp, int]:
         ry = ResourceRY.resource_rep()
         rz = ResourceRZ.resource_rep()
 
-        gate_types = {rx: 1, ry: 1, rz: 1}
+        gate_types = {ry: 1, rz: 2}
```

Comment:
Why is this change needed?

Reply:
I think it was just implemented incorrectly?